cluster/gce/gci/configure-helper-common.sh

```diff
@@ -2347,7 +2347,7 @@ function start-kube-controller-manager {
     RUN_CONTROLLERS="serviceaccount,serviceaccount-token,nodelifecycle,ttl,daemonset,csrsigning,csrapproving,csrcleaner"
   fi
   if [[ "${ARKTOS_SCALEOUT_SERVER_TYPE:-}" == "tp" ]]; then
-    RUN_CONTROLLERS="*,-nodeipam,-nodelifecycle,-mizar-controllers,-network,-ttl,-daemonset"
+    RUN_CONTROLLERS="*,-nodeipam,-nodelifecycle,-mizar-controllers,-network,-ttl"
```

Comment:
please remove the daemonset controller from the RP as well with this change to make it complete.

Reply:
Do you mean we don't need Daemonset for RP? we need double check this. For now, local environments has both enabled Daemonset.

Reply:
In theory, Daemonset controller should not be running in RP. I will take it off and test in local env.

Reply:
I have tested in local dev env, daemonset controller can be disabled for RP.

Reply:
removed